etcdctl/ctlv3/command/ep_command.go

```diff
@@ -81,15 +81,28 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 	}
 
 	var wg sync.WaitGroup
+	errc := make(chan error, len(cfgs))
+	errCountc := make(chan int, 1)
+	defer close(errCountc)
+
+	// drain errors channel until it is closed.
+	go func() {
```

Comment:
this extra concurrency isn't necessary since `errc` is already buffered-- keep it simple by doing the error pass after the other goroutine complete:
```go
wg.Wait()
close(errc)
errs := false
for err := range errc {
    if err != nil {
        errs = true
        fmt.Fprintln(os.Stderr, err)
    }   
}
if errs {
    ExitWithError(ExitError, fmt.Errorf("unhealthy cluster"))
}
```